specification/monitor/resource-manager/readme.python.md

```diff
@@ -231,16 +231,6 @@ namespace: $(python-base-namespace).v2017_12_01_preview
 output-folder: $(python-sdks-folder)/$(python-base-folder)/v2017_12_01_preview
 ```
 
-### Tag: package-2017-11-01-preview-only and python
-
-These settings apply only when `--tag=package-2017-11-01-preview-only --python` is specified on the command line.
-Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
-
-``` yaml $(tag) == 'package-2017-11-01-preview-only'
```

Comment:
you still have it `package-2017-11-01-preview-only` in readme. Just search it.

Reply:
@weidongxu-microsoft sorry for missing that. Now all package-2017-11-01-preview-only are removed from that file